pkg/controllers/clusterregistration/controller.go

```diff
@@ -160,9 +160,6 @@ func (h *handler) authorizeCluster(sa *v1.ServiceAccount, cluster *fleet.Cluster
 			Labels: map[string]string{
 				fleet.ClusterAnnotation: cluster.Name,
 			},
-			Annotations: map[string]string{
-				fleet.ManagedLabel: "true",
```

Comment:
Same as above. This secret will not be purged, maybe it was put into the annotations list by accident? Are we leaving cluster registration secrets behind?